src/kitovu/sync/syncplugin.py

```diff
@@ -12,6 +14,8 @@ class AbstractSyncPlugin(metaclass=abc.ABCMeta):
     Every abstract method in this class is a plugin hook.
     """
 
+    reporter: utils.AbstractReporter
```

Comment:
This is a class instead of instance variable, so this should be set in `__init__` (also see my comment above)